pack and install: should be able to webpack the library failed

Note: #1877 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 08978822e1b7b5961f0e355df51d738e012be392
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/5e3c8532-8d2f-45d4-ab30-997af11fe5a0), [Sponge](http://sponge2/5e3c8532-8d2f-45d4-ab30-997af11fe5a0)
status: failed

Ah! Yes, here:

```
found 5 vulnerabilities (2 moderate, 3 high)
  run `npm audit fix` to fix them, or `npm audit` for details
Hash: 27080731c59816c644f5
Version: webpack 4.47.0
Time: 3208ms
Built at: 11/19/2024 9:35:46 AM
                  Asset      Size  Chunks  Chunk Names
../build/src/index.d.ts  11 bytes          
          bundle.min.js  4.81 KiB       0  main
Entrypoint main = bundle.min.js
[0] ./src/index.ts 1.08 KiB {0} [built]

ERROR in ./src/index.ts
Module not found: Error: Can't resolve 'google-auth-library' in '/tmp/google-auth-library-nodejs-pack-GiNh0c/src'
 @ ./src/index.ts 16:30-60

ERROR in bundle.min.js from Terser
Error: Cannot find module '../package.json'
```

It seems like something is packing the auth library and trying to install it, but it's not actually packed and available, so it seems like maybe there needs to be more time to allow the library to be installed before tests are run. But, I didn't spend too much time on it, so feel free to investigate it further and come up with whatever solution you think is best!

--- a/system-test/test.kitchen.ts
+++ b/system-test/test.kitchen.ts
@@ -31,6 +31,11 @@ const pkg = require('../../package.json');
 let stagingDir: string;
 
 /**
+ * 2 minutes
+ */
+const BUILD_TEST_TIMEOUT_MS = 2 * 60_000;
+
+/**
  * Spawns and runs a command asynchronously.
  *
  * @param params params to pass to {@link spawn}
@@ -87,13 +92,15 @@ describe('pack and install', () => {
     // npm, once in a blue moon, fails during pack process. If this happens,
     // we should be safe to retry.
     this.retries(3);
-    this.timeout(40000);
+    this.timeout(BUILD_TEST_TIMEOUT_MS);
+
     await packAndInstall();
   });
 
   it('should be able to webpack the library', async function () {
     this.retries(3);
-    this.timeout(40000);
+    this.timeout(BUILD_TEST_TIMEOUT_MS);
+
     await packAndInstall();
     // we expect npm install is executed in the before hook
     await run('npx', ['webpack'], {cwd: `${stagingDir}/`});

